### 1.stage
> a,Schematic of mouse spermatogenesis and the four representative stages analyzed in this study: THY1+, undifferentiated spermatogonia; KIT+, differentiating spermatogonia; PS, pachytene spermatocytes; RS, round spermatids.

<img width="553" alt="截屏2024-08-27 18 42 02" src="https://github.com/user-attachments/assets/2ab1c227-6c94-4b30-9ffa-8dfafed0bc75">

(https://www.nature.com/articles/s41594-020-0487-4)

Fig.1a

### 2.gene structure
> b, Schematic for the generation of a high-confidence, ‘best match’ TE annotation set (n = 1,755,061). RepeatMasker annotation was used to specify the best-matched repetitive element class/family for a given locus. We removed TEs overlapping exons and/or poorly matching the consensus sequence of their specified element class, as evidenced by a low Smith–Waterman (SW) alignment score. 

<img width="237" alt="截屏2024-08-27 18 43 51" src="https://github.com/user-attachments/assets/e8ab4475-2030-4e97-97a3-af6f7853488f">


(https://www.nature.com/articles/s41594-020-0487-4)

Fig.1b

### 3.pipeline

> a, The RNA-seq pipeline for comprehensive quantification of TE copies. The flowchart indicates the various RNA-seq and data analysis processes that comprise the pipeline. Round-corner rectangles, input files; rectangles, output files; diamond, branch condition. The specific tools used are highlighted in red. 

<img width="477" alt="截屏2024-08-27 18 49 39" src="https://github.com/user-attachments/assets/cfe63920-f01d-484c-b43c-0a3791649c65">

(https://www.nature.com/articles/s41594-020-0487-4)

Extended Data Fig.1
